mix.exs

```diff
@@ -65,4 +75,24 @@ defmodule Wallaby.Mixfile do
       main: "readme",
     ]
   end
+
+  defp test_paths(driver) when driver in @drivers, do: ["integration_test/#{driver}"]
+  defp test_paths(_), do: ["test"]
+
+  defp test_drivers(args) do
+    for driver <- @drivers, do: run_integration_test(driver, args)
+  end
+
+  defp run_integration_test(driver, args) do
+    args = if IO.ANSI.enabled?, do: ["--color"|args], else: ["--no-color"|args]
```

Comment:
minor nitpick - maybe we should get credo or something, like space around `|` :)